docs/config/README.md

```diff
@@ -46,9 +46,10 @@ This is the list of prompt-wide configuration options.
 
 ### Options
 
-| Variable      | Default | Description                                    |
-| ------------- | ------- | ---------------------------------------------- |
-| `add_newline` | `true`  | Add a new line before the start of the prompt. |
+| Variable       | Default | Description                                                        |
+| -------------- | ------- | ------------------------------------------------------------------ |
+| `add_newline`  | `true`  | Add a new line before the start of the prompt.                     |
+| `prompt_order` | `[]`    | Configure out custom prompt order overriding default prompt order. |
```

Comment:
I think we may want to add the default `prompt_order` here, but it may be too big a list for the table.
Perhaps we could add a separate subb-heading to the "Prompt" heading specifically detailing the default prompt order and how changing it would work.

Reply:
@matchai That was originally in the PR, but I voiced concerns about having that exposed separately (it means that we'd have to mirror changes in this doc as well every time we make a change, and we've been making a lot recently, with two or three PRs in the last 24 hours that would have needed to remember to make changes there).

Would it be possible to insert something here that generates the order directly from the code, so that we don't have to keep them in sync?

Reply:
Perhaps a really simple solution would be to include a link to the GitHub source code (with the correct line number) for anyone who wants to know what the default order is.

Reply:
> Perhaps a really simple solution would be to include a link to the GitHub source code (with the correct line number)

I'm in favor of this 👍

Reply:
Main issue for linking out line number is changing line number since above `DEFAULT_PROMPT_ORDER` we have `ALL_MODULE` const and import statement which needs to be updated out for each addition of module since each modules need to be added over `ALL_MODULE` for custom prompt order to work as well as we may add new imports. Maybe adding subheading as orginally in the PR and commenting `DEFAULT_PROMPT_ORDER` code with information that if someone change this value they need to also update config docs

Reply:
I think that's reasonable. 👍